examples/pipelines/simple_pipeline/components/load_from_hub/requirements.txt

```diff
@@ -1,4 +1,4 @@
-datasets==2.11.0
+huggingface_hub==0.14.1
```

Comment:
Do we need this? I don't see this used explicitly in the code.

Reply:
This is needed for the HF filesystem

Reply:
Yes but how is it used if we are not calling it directly? Is it a subdependency of another package?

Reply:
It's a subdependency of `fsspec`, see https://huggingface.co/docs/huggingface_hub/main/package_reference/hf_file_system